### DISCUSSION TITLE
What about Stylelint?

### DISCUSSION THREAD

**Discussion by firefoxic** [eslint-stylistic/eslint-stylistic#192] [General]
Hi!
Is it possible to somehow join your `@stylistic` organization in `npm` with my [`stylelint-codeguide`](https://github.com/firefoxic/stylelint-codeguide) plugin (I don't seem to have named it well), with the new name `@stylistic/stylelint-plugin`? Seems like it would make it easier for those who don't fit into `prettier` to find the full set of linter plugins this way.

**Comment by kecrily** [comment]
@antfu What do you think?
